README.md

```diff
@@ -4,7 +4,7 @@
 Add following step to your workflow:
 
 ```yaml
-- uses: dev-drprasad/delete-tag-and-release@v0.2.1
+- uses: dev-drprasad/delete-tag-and-release@v0.2.1 # PRERELEASE is v1.0 and can also be used to test and give us feedback
```

Comment:
Sorry for my lateness; I totally forgot about this ticket!

But we actually want to do the opposite. The comment is good, but we still want to have the uses version be 1.0; otherwise users will copy this block expecting to use 1.0 (because that's the version they chose on the GitHub Actions page) and instead get 0.2.1.